tests/e2e/test_rosacli_ingress.go

```diff
@@ -145,6 +145,23 @@ var _ = Describe("Edit default ingress",
 						"ERR: Updating route selectors is not supported for Hosted Control Plane clusters"))
 			})
 
+		It("can describe ingress of a cluster - [id:73538]",
+			labels.Low, labels.Runtime.Day2,
+			func() {
+				By("Retrieve cluster and get default ingress id")
+				output, err := ingressService.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				ingressList, err := ingressService.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				var defaultID string
+				for _, v := range ingressList.Ingresses {
+					defaultID = v.ID
+				}
+				_, err = rosaClient.Ingress.DescribeIngressAndReflect(clusterID, defaultID)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
You should check the output of the describe ingress command here

Reply:
There are two more checkpoints in the test case about wrong cluster id and wrong ingressID provided, I guess it can be added here

Reply:
Added the output check. Moved the last 2 checkpoints into a negative test case (75052)